### PR TITLE
Add Platform.Converters/0.1.0

### PR DESCRIPTION
**Pull request by Mitron57**
**platform.converters/0.1.0**
---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/platform.converters/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class PlatformConvertersConan(ConanFile):
+    name = "platform.converters"
```
Comment (uilianries) [inline recipes/platform.converters/all/conanfile.py:9]: The name is a bit out of is used in Conan Center, but I'm fine as it's already used on nuget with the same name.

File: recipes/platform.converters/all/test_package/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
```
Comment (uilianries) [inline recipes/platform.converters/all/test_package/CMakeLists.txt:8]:
```suggestion
conan_basic_setup(TARGETS)

find_package(platform.converters CONFIG REQUIRED

add_executable(${PROJECT_NAME} test_package.cpp)
target_link_libraries(${PROJECT_NAME} platform.converters::platform.converters)
```
CONAN_LIBS will be deprecated soon.

File: recipes/platform.converters/all/test_package/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
```
Comment (uilianries) [inline recipes/platform.converters/all/test_package/conanfile.py:7]:
```suggestion
    generators = "cmake", "cmake_find_package_multi"
```
it generates platform.converters-config.cmake

**Review by uilianries — COMMENTED** [review]
ONly need to update the test package and we are good to go.

File: recipes/platform.converters/all/test_package/CMakeLists.txt (hunk 2 above)
Comment (Mitron57) [reply recipes/platform.converters/all/test_package/CMakeLists.txt:8]: Ok, I'll change it to yours

**Review by uilianries — APPROVED** [review]
LGTM

**Review by SSE4 — APPROVED** [review]

**Pull request merged**